src/data/matrix/basic.lean

```diff
@@ -353,6 +353,54 @@ lemma map_mul {L : matrix m n α} {M : matrix n o α}
 (L ⬝ M).map f = L.map f ⬝ M.map f :=
 by { ext, simp [mul_apply, ring_hom.map_sum], }
 
+/-- A version of `one_map` where `f` is a ring hom. -/
```

Comment:
Perhaps:

```suggestion
/-!
Variants of `one_map` and `zero_map` for each of the many bundled homs and equivs.
This hopefully won't be necessary [in future](https://leanprover.zulipchat.com/#narrow/stream/113488-general/topic/Typeclasses.20for.20morphisms)
-/

/-- A version of `one_map` where `f` is a ring hom. -/
```
or some other "this is a mess that we know about and should be discussed here" comment.